semantic-conventions/README.md

```diff
@@ -2,6 +2,23 @@
 
 A docker image to process Semantic Convention YAML models.
 
+## Development
+
+**Build Dependencies**:
+
+- pip
+- docker
```

Comment:
You could mention that docker is only needed if you actually want to build the docker image. You can build & run the semantic convention generator just with Python.

Reply:
If we put it on a CONTRIBUTING, then docker is removed from the equation.